implementation.js

```diff
@@ -0,0 +1,27 @@
+/*! https://mths.be/repeat v1.0.0 by @mathias */
+
+'use strict';
+
+var RequireObjectCoercible = require('es-abstract/2019/RequireObjectCoercible');
+var ToString = require('es-abstract/2019/ToString');
+var ToInteger = require('es-abstract/2019/ToInteger');
+
+module.exports = function repeat(count) {
+	var O = RequireObjectCoercible(this);
+	var string = ToString(O);
+	var n = ToInteger(count);
+	// Account for out-of-bounds indices
+	if (n < 0 || n == Infinity) throw RangeError();
```

Comment:
would be nice to use a meaningful error message here; here's Safari's:
```suggestion
	if (n < 0 || n == Infinity) throw RangeError('String.prototype.repeat argument must be greater than or equal to 0 and not be Infinity');
```

Reply:
I'm trying to keep the polyfill behavior the same in these refactoring PRs, but I can do that as a follow up!

Reply:
Fair, although since it's a breaking change PR that's not necessarily a critical goal ;-)